two-step-view/README.md

```diff
@@ -0,0 +1,79 @@
+# Book Data Processing and Presentation
```

Comment:
`README.md` has a very specific format that needs to be implemented by all the pattern examples. See here: https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute